test/e2e/pipeline/start_test.go

```diff
@@ -0,0 +1,183 @@
+					return err
+				}
+
+				if _, err := c.ExpectString("skaffold-git (https://github.com/GoogleContainerTools/skaffold)"); err != nil {
+					return err
+				}
+
+				if _, err := c.SendLine(string(terminal.KeyEnter)); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectEOF(); err != nil {
+					return err
+				}
+
+				c.Close()
+				return nil
+			}})
+
+	})
+
+	t.Run("Validate interactive pipeline logs, with  follow mode (-f) ", func(t *testing.T) {
+		tkn.RunInteractiveTests(t, &e2e.Prompt{
+			CmdArgs: []string{"pipeline", "logs", "-f"},
+			Procedure: func(c *expect.Console) error {
+				if _, err := c.ExpectString("Select pipeline:"); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("output-pipeline"); err != nil {
+					return err
+				}
+
+				if _, err := c.Send(string(terminal.KeyEnter)); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectEOF(); err != nil {
+					return err
+				}
+
+				c.Close()
+				return nil
+			}})
+	})
+}
+
+func TestPipelineInteractiveStartWithNewResourceE2E(t *testing.T) {
+	t.Parallel()
+	c, namespace := e2e.Setup(t)
+	knativetest.CleanupOnInterrupt(func() { e2e.TearDown(t, c, namespace) }, t.Logf)
+	defer e2e.TearDown(t, c, namespace)
+
+	kubectl := e2e.NewKubectl(namespace)
+	tkn, err := e2e.NewTknRunner(namespace)
+	if err != nil {
+		t.Fatalf("Error creating tknRunner %+v", err)
+	}
+
+	t.Logf("Creating pipeline in namespace: %s", namespace)
+	e2e.Assert(t, kubectl.Create(e2e.ResourcePath("pipeline.yaml")), icmd.Success)
+
+	t.Run("Start PipelineRun using pipeline start interactively with SA as 'pipeline' ", func(t *testing.T) {
+		tkn.RunInteractiveTests(t, &e2e.Prompt{
+			CmdArgs: []string{"pipeline", "start", "output-pipeline", "-s", "pipeline"},
+			Procedure: func(c *expect.Console) error {
+				if _, err := c.ExpectString("Please create a new \"git\" resource for pipeline resource \"source-repo\""); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("Enter a name for a pipeline resource :"); err != nil {
+					return err
+				}
+
+				if _, err := c.SendLine("skaffold-git"); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("Enter a value for url :"); err != nil {
+					return err
+				}
+
+				if _, err := c.SendLine("https://github.com/GoogleContainerTools/skaffold"); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("Enter a value for revision :"); err != nil {
+					return err
+				}
+
+				if _, err := c.SendLine("master"); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectString("New git resource \"skaffold-git\" has been created"); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectEOF(); err != nil {
+					return err
+				}
+
+				c.Close()
+				return nil
+			}})
+
+	})
+
+	t.Run("Validate interactive pipeline logs, with  follow mode (-f) ", func(t *testing.T) {
```

Comment:
Can this include actually selecting and starting the logs of the PipelineRun?

Reply:
yes but we need at least 2 pipelineruns for same pipeline I guess I could add this as another scenario if it works

Reply:
Sure, sounds good to me.